app/forms/signals_telegram_bot_registration_form.rb

```diff
@@ -8,7 +7,7 @@ class SignalsTelegramBotRegistrationForm < Patterns::Form
 
   validates :telegram_username, presence: true
   validates :user, presence: true, :if => proc { |f| f.telegram_username.present? }
-  validate :validate_user_staked_cofi_amount, :if => proc { |f| f.user.present? }
+  validate :validate_user_access, :if => proc { |f| f.user.present? }
```

Comment:
Can we reorder this to the end of the `validates` lines to make it more clear that this particular validation is referring to a method instead of a model field?

Reply:
Sure, moving it definitely a lot clearer